src/classes/OPP_OpportunityContactRoles_TEST.cls

```diff
@@ -638,4 +638,75 @@ private class OPP_OpportunityContactRoles_TEST {
         System.assertEquals(oppty.Primary_Contact__c, ocrsUpdated[0].ContactId);
     }
 
+
+    /*********************************************************************************************************
+    * @description Tests retrieving the Opportunity's associated Account's details.
+    * verify:
+    *   The Opportunity's Account details are retrieved.
+    * @return void
+    *********************************************************************************************************/
+    static testMethod void testPopulateAccountDetails() {
```

Comment:
if we are going to invest is unit testing of specific methods, why not do it with bulk data to ensure it truly works?

Reply:
We definitely do that.